clang/lib/AST/ByteCode/Boolean.h

```diff
@@ -81,6 +81,14 @@ class Boolean final {
 
   Boolean truncate(unsigned TruncBits) const { return *this; }
 
+  static Boolean bitcastFromMemory(const std::byte *Buff, unsigned BitWidth) {
+    assert(BitWidth == 8);
```

Comment:
Perhaps a `//FIXME: <why this is limited to 8 bits, and what we have to do to fix this>` here.